pyslackers_website/static/css/member_map.css

```diff
@@ -0,0 +1,6 @@
+#leaflet_member_map {
```

Comment:
This won't be very responsive, can the available classes from semantic work for this so it responds and collapses?